os/fs/littlefs/littlefs/lfs.c

```diff
@@ -6570,7 +6570,7 @@ int lfs_check_format(lfs_t *lfs, const struct lfs_config *cfg)
 	}
 cleanup:
 	lfs_deinit(lfs);
-    LFS_UNLOCK(lfs->cfg);
+    LFS_UNLOCK(cfg);
```

Comment:
please check indentation

Reply:
this is littlefs code, not ours I will keep this until it become stable we pulled 2 times from littlefs github already.